src/test/java/io/vertx/test/core/HttpTest.java

```diff
@@ -129,6 +139,17 @@ public void testClientChaining() {
     await();
   }
 
+  @Test
+  public void testListenSocketAddress() {
+    server = vertx.createHttpServer().requestHandler(noOpHandler());
+    server.listen(testAddress, onSuccess(server -> {
+      testComplete();
+    }));
+
```

Comment:
we should also have a client and complete the test when the server gets the request